packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDeploySourcePath.test.ts

```diff
@@ -8,15 +8,17 @@
 import { AuthInfo, Connection } from '@salesforce/core';
 import { MockTestOrgData, testSetup } from '@salesforce/core/lib/testSetup';
 import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types/index';
-import { ComponentSet, MetadataResolver } from '@salesforce/source-deploy-retrieve';
+import {
+  ComponentSet,
+  MetadataResolver
+} from '@salesforce/source-deploy-retrieve';
 import { expect } from 'chai';
 import * as path from 'path';
 import { createSandbox, SinonStub } from 'sinon';
-import {
-  LibraryDeploySourcePathExecutor
-} from '../../../src/commands';
+import * as vscode from 'vscode';
+import { LibraryDeploySourcePathExecutor } from '../../../src/commands';
+import * as forceSourceDeploySourcePath from '../../../src/commands/forceSourceDeploySourcePath';
```

Comment:
Just a FYI... I ran the liter and it applied these formatting changes.  It wasn't something I explicitly did.